HW1/P6/P6.py

```diff
@@ -0,0 +1,35 @@
+import findspark
+findspark.init()
+import pyspark
+sc = pyspark.SparkContext(appName="")
+import re
+import random
+
+## Note that to I have excluded the licensen information from the downloaded for text for better accuracy
+## Pre-process the text to create a list of words without all caps, all caps with an ending period, and all nums
```

Comment:
I don't think that this was necessary but it seems like a great idea as you get a more accurate model of Shakespeare's writing.